css/admin-page.styl

```diff
@@ -0,0 +1,13 @@
+.admin-page
+  .project-status
```

Comment:
With BEM none of these need to be nested.

You could still imply the hierarchy with the Stylus `&` feature:

```styl
.project-status
  &__section
    margin: ...

    &--list
      margin: ...
```